src/main/resources/component-biz.conf

```diff
@@ -23,6 +23,29 @@ journal {
     keep-incidents = 30 days
 }
 
+health.limits {
+    # Controls how many statistics updates per minute are considered healthy
+    # The actual values depend heavily on the system and application - these
+    # just seem like reasonable standard values.
+    statistics-updates.gray  =  10
+    statistics-updates.warn  = 500
+    statistics-updates.error =   0
+
+    # If there is any lock held, we will report this - but there is no
+    # sane limit how many locks can be considered healthy / unhealty
+    locks-count.gray  = 1
+    locks-count.warn  = 0
+    locks-count.error = 0
+
+    # We start to warn as soon as we encounter one long running lock
+    # (held for at least 30min). As this can still be quite alright
+    # we do not consider this critical (red)
+    locks-long-running.gray  = 0
+    locks-long-running.warn  = 1
+    locks-long-running.error = 0
+}
```

Comment:
collector.metric("locks-long-running", "Long locks", numberOfLongRunningLocks, null);
in Locks.java:171